modules/ocf/manifests/hidepid.pp

```diff
@@ -0,0 +1,36 @@
+class ocf::hidepid {
+
+  $group_cap_ptrace = 'capptrace'
+
+  group { $group_cap_ptrace:
+    ensure => present,
+    name   => $group_cap_ptrace,
+    system => true,
+    before => Mount['/proc'];
+  } ->
+
+  ocf::systemd::override { 'hidepid-logind':
+    unit    => 'systemd-logind.service',
+    content => "[Service]\nSupplementaryGroups=${group_cap_ptrace}\n",
+  }
+
+  if str2bool($::polkit_installed) {
+    # NOTE: Futureproof for policykit for version 0.115. Since polkit < 0.115
+    # runs as root this change incurs no security cost.
+    ocf::systemd::override { 'hidepid-polkit':
```

Comment:
I feel like having this unconditionally might be better, unless there is really a reason we must check for existence of PolKit

Reply:
This will crash on death for example, because `polkit` is not installed there. Some other package is pulling it in that is not present on all of our systems.

Reply:
I have a hunch that `polkit` is only on systems with `cups`. After poking around with some rdepends on supernova, it looks like `colord` is the only hard dependency on `polkit`, and `cups` on `colord`.